ImageD11/sandbox/cryst_ref_frames.ipynb

```diff
@@ -0,0 +1,1112 @@
+{
```

Comment:
<p>WIthin a cif file people will find <code>atom_site_fract_[xyz]</code> and<code>atom_site_cartn_[xyz]</code>   </p><p>People use 'xyz' for these numbers (e.g. EXYZ shelx instruction). </p><p> </p><p>I don't remember reading [uvw] for co-ordinates of atoms.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/FABLE-3DXRD/ImageD11/pull/439/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>